diff --git a/PcapPlusPlus-24.09/Packet++/header/DhcpLayer.h b/PcapPlusPlus-24.09/Packet++/header/DhcpLayer.h
index 435becb..282d739 100644
--- a/PcapPlusPlus-24.09/Packet++/header/DhcpLayer.h
+++ b/PcapPlusPlus-24.09/Packet++/header/DhcpLayer.h
@@ -881,8 +881,8 @@ namespace pcpp
 
 	bool DhcpLayer::isDhcpPorts(uint16_t portSrc, uint16_t portDst)
 	{
-		return ((portSrc == 68 && portDst == 67) || (portSrc == 67 && portDst == 68) ||
-		        (portSrc == 67 && portDst == 67));
+		// src port can be any ephemeral port so removing the check
+		return ((portDst == 67) || (portDst == 68));
 	}
 
 }  // namespace pcpp
